paginator/paginator.go

```diff
@@ -81,7 +81,7 @@ func (m *Model) SetTotalPages(items int) int {
 	return n
 }
 
-// ItemsOnPage is a helper function for returning the numer of items on the
+// ItemsOnPage is a helper function for returning the number of items on the
 // current page given the total numer of items passed as an argument.
```

Comment:
The same typo is repeated in this line, as well as in `table/table.go`. Would be great to fix them all at once.

Reply:
Right.
I fixed them too.